.env.test

```diff
@@ -1,4 +1,5 @@
 PORT=3000
 NODE_ENV=testing
 JWT_SECRET=thisismysupersecrettokenjustkidding
-DATABASE_URL=mongodb+srv://donut-admin:5cdS2C2g3wRAdQWp@donut-users-hdawt.mongodb.net/donut-testing?retryWrites=true&w=majority
\ No newline at end of file
+DATABASE_URL=mongodb+srv://donut-admin:5cdS2C2g3wRAdQWp@donut-users-hdawt.mongodb.net/donut-testing?retryWrites=true&w=majority
+SENDGRID_API_KEY=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
```

Comment:
this can be misused

Reply:
I didn't realize I have pushed the keys. I will remove them. Thanks for pointing put @vaibhavdaren